<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionInterface</class>
 <widget class="QWidget" name="SelectionInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diffusion</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,1">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Lundi de la semaine :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="edit_monday">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="list_mondays">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="list_classes">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
